nlp/main.go

```diff
@@ -0,0 +1,51 @@
+package nlp
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+
+	"github.com/dulwin/ezebot/utils"
+)
+
+type WitResponse struct {
+	MessageId string              `json:"msg_id"`
+	Text      string              `json:"_text"`
+	Entities  map[string][]Entity `json:"entities"`
+}
+
+type Entity struct {
+	Confidence float32 `json:"confidence"`
+	Value      string  `json:"value"`
+	Type       string  `json:"type,omitempty"`
+	Grain      string  `json:"grain,omitempty"`
+}
+
+const URL string = "https://api.wit.ai/message?"
+
+func ProcessMessage(message string) *WitResponse {
+
+	client := &http.Client{}
+	queryString := url.Values{}
+	queryString.Set("q", message)
+
+	requestUrl := URL + queryString.Encode()
```

Comment:
I think this is fine for now; however, for future use we might end up integrating further with wit (maybe?) and creating our own wit http client might be handy. However, I don't see that happening any time soon. Probably just something to keep in mind.

i.e.

``` go
witResponse := witClient.Get(message_body) 
```